happytransformer/happy_transformer.py

```diff
@@ -76,3 +76,19 @@ def test(self, input_filepath, args):
 
         """
         raise NotImplementedError()
+
+    def _create_args_dataclass(self, default_dic_args, input_dic_args, dataclass_args):
+        """
+        Combines default_dic_args and input_dic_args and then outputs a dataclass.
+
+        The values of input_dic_args overwrite the values of default_dic_args.
+
+        default_dic_args and dataclass_args must have the same keys.
+
+        :param default_dic_args: A dictionary that contains default settings. Example: ARGS_WP_EVAl
+        :param input_dic_args: A dictionary a user inputs for **kwargs when using .train(), .eval() or .test()
+        :param dataclass_args: A dataclass object for target functionality. Example: WPEvalArgs
+        :return: A dataclass object that will then be passed to HappyTrainer.train()/eval/test
```

Comment:
Naming between `default_dic_args` and `dataclass_args` is waaaay too similar, given that they represent entirely different things (dictionary vs class type)

Reply:
yeah, thinking of good names it tricky since all three inputs are fairly similar. I changed it to "dataclass_args" to "method_dataclass_args" in 4996efa.

default_dic_args and method_dataclass_args are only used internally, so we can easily change them in the future without it resulting in any breaking changes.

Reply:
Wait a second. I think this entire method might be redundant. Will elaboarate later